float64 leftSpeed
float64 rightSpeed
float64 liftThrust
